requirements.txt

```diff
@@ -6,3 +6,5 @@ flake8==3.9.2
 pylint==2.8.2
 pytest==6.2.4
 pytest-cov==2.11.1
+sphinx~=4.0.0
+sphinx-autoapi~=1.8.1
```

Comment:
```suggestion
sphinx==4.0.0
sphinx-autoapi==1.8.1
```

Reply:
Hang on, I forgot something, will fix it in a sec.